lib/cartopy/tests/test_shapereader.py

```diff
@@ -83,7 +83,8 @@ class TestRivers:
     def setup_class(self):
         RIVERS_PATH = shp.natural_earth(resolution='110m',
                                         category='physical',
-                                        name='rivers_lake_centerlines')
+                                        name='rivers_lake_centerlines',
+                                        version='5.0.0')
```

Comment:
Similarly, did this specific version (`5.0.0`) of rivers pass the image comparison tests?

Reply:
Yes, it was passing. Note there is no image comparison here, the test just looks at a few very specific features and there was no change.